pkg/actionsmetrics/event_reader.go

```diff
@@ -137,7 +137,11 @@ func (reader *EventReader) ProcessWorkflowJobEvent(ctx context.Context, event in
 		githubWorkflowJobQueueDurationSeconds.With(labels).Observe(parseResult.QueueTime.Seconds())
 
 	case "completed":
-		githubWorkflowJobsCompletedTotal.With(extraLabel("runner_name", *e.WorkflowJob.RunnerName, labels)).Inc()
+		runnerName := ""
+		if e.WorkflowJob.RunnerName != nil {
+			runnerName = *e.WorkflowJob.RunnerName
+		}
+		githubWorkflowJobsCompletedTotal.With(extraLabel("runner_name", runnerName, labels)).Inc()
```

Comment:
maybe let's not add it when it's nill or empty? and add it inside if?

Reply:
adjusted 👍🏼 also followed the nil check convention from other parts of the event reader

Reply:
looks like this causes an issue due to cardinality being inconsistent.